packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -790,8 +790,8 @@ class CupertinoDynamicColor extends Color with DiagnosticableMixin implements Di
   ///
   /// In other words, this color will be the effective color of the [CupertinoDynamicColor]
   /// after it is resolved against a [BuildContext] that:
-  /// - has a [CupertinoTheme] whose [brightness] is [PlatformBrightness.light],
-  /// or a [MediaQuery] whose [MediaQueryData.platformBrightness] is [PlatformBrightness.light].
+  /// - has a [CupertinoTheme] whose [CupertinoThemeData.brightness] is [Brightness.light],
```

Comment:
Is the only difference from mine the `CupertinoThemeData.brightness`?

Will this just cause merge conflicts?

Reply:
Whoops I thought you only changed one of them. Yes that's only difference. I'll wait for your PR to get merged in case there're merge conflicts

Reply:
conflicts resolved